lib/commands/run.rb

```diff
@@ -2,16 +2,16 @@
 require_relative './compose'
 
 class Commands::Run < Commands::Base
-  def initialize(stack, args, service = nil, config_directory = nil)
-    super(service, config_directory, nil, stack)
+  def initialize(stack, verbose, args, service = nil, config_directory = nil)
```

Comment:
Minor, but this is probably a good time to start using hash args :-).

Reply:
I agree. I think the whole base command initialiser could do with a refactor now anyway which I'm planning to do in a later PR, so we can merge this one in for now I think.